app.js

```diff
@@ -2,16 +2,37 @@ const express = require("express");
 const app = express();
 const {getAllTopics} = require("./controllers/topics_controllers")
 const {getAllEndpoints} = require("./controllers/endpoints_controllers")
+const {getArticleById} = require("./controllers/articles_controllers")
 
 app.use(express.json());
```

Comment:
Do you need this in your code yet? Remove this line until you need it (i.e. post / patch)